oauthproxy.go

```diff
@@ -667,6 +678,24 @@ func (p *OAuthProxy) SignOut(rw http.ResponseWriter, req *http.Request) {
 func (p *OAuthProxy) OAuthStart(rw http.ResponseWriter, req *http.Request) {
 	prepareNoCache(rw)
 
+	params := mux.Vars(req)
+	idString := (params["id"])
+
+	var providerSlice int
+	var ok bool
+
+	if idString == "" {
```

Comment:
Perhaps we need an option to specify the default providerID somewhere within the config so that it's explicit rather than relying on the first option